src/main/code.ts

```diff
@@ -116,7 +119,7 @@ if (figma.currentPage.selection.length != 1) {
   
   function getFirstChildOfTypeText(sceneNode: SceneNode) {
 	if ("children" in sceneNode) {
-	  return sceneNode.children.find(node => node.type === "TEXT") as TextNode
+	  return sceneNode.findOne(node => node.type === "TEXT") as TextNode
 	} else {
 	  return undefined
```

Comment:
To simplify check in calling code

```suggestion
	  return null
```